engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/NadelExecutionContext.kt

```diff
@@ -0,0 +1,12 @@
+package graphql.nadel.enginekt
+
+import graphql.ExecutionInput
+
+data class NadelExecutionContext(
```

Comment:
We now pass this around instead of just `userContext`.

The `executionInput` is required to call the execution engine again.